static/src/stylesheets/module/email/_form.scss

```diff
@@ -7,6 +7,11 @@
     // adjustments, off grid
     width: calc(100% + 8px);
     margin-left: -4px;
+
+    @include mq($from: mobileLandscape, $until: tablet) {
+        margin-right: 50%;
+        width: calc(50% + 8px);
```

Comment:
Hmm, this is a good point and makes sense in terms of the layout passes. Do you have any reading on this? I couldn't find anything in-particular RE calc performance, though obviously multiple passes on the layout is a common sense performance hit... I'll have a re-think anyway.